src/algebraic_topology/dold_kan/compatibility.lean

```diff
@@ -176,6 +176,119 @@ end
 
 lemma equivalence_functor : (equivalence hF hG).functor = F ⋙ eB.inverse := rfl
 
+omit hG hF
+
+/-- The isomorphism `eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor` deduced
+from the counit isomorphism of `e'`. -/
+@[simps hom_app]
+def τ₀ : eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor :=
+calc eB.functor ⋙ e'.inverse ⋙ e'.functor
+  ≅ eB.functor ⋙ 𝟭 _ : iso_whisker_left _ e'.counit_iso
+... ≅ eB.functor : functor.right_unitor _
+
+include hF hG
+
+/-- The isomorphism `eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor` deduced
+the isomorphisms `hF : eA.functor ⋙ e'.functor ≅ F`,
+`hG : eB.functor ⋙ e'.inverse ≅ G ⋙ eA.functor` and the datum of
+an isomorphism `η : G ⋙ F ≅ eB.functor`. -/
+@[simps hom_app]
+def τ₁ (η : G ⋙ F ≅ eB.functor) :
+  eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor :=
+calc eB.functor ⋙ e'.inverse ⋙ e'.functor
+  ≅ (eB.functor ⋙ e'.inverse) ⋙ e'.functor : iso.refl _
+... ≅ (G ⋙ eA.functor) ⋙ e'.functor : iso_whisker_right hG _
```

Comment:
At the very least my instinct is to get the isomorphisms to all line up vertically.